iosxr_store_box.py

```diff
@@ -48,7 +48,7 @@ def generate_hash(file):
     # SHA256 the box file and store in same location
     sha256_hash = hashlib.sha256(open(file, 'rb').read()).hexdigest()
     logger.debug('SHA256: %s' % sha256_hash)
-    hash_file = os.path.splitext(file)[0] + '.sha256.txt'
+    hash_file = os.path.splitext(file)[0] + '.box.sha256.txt'
```

Comment:
Yes it is:

rwellum@RWELLUM-M-34DF:[~/Desktop/Boxes]: iosxrv-x64-vbox/iosxr_store_box.py machines/iosxrv-fullk9-x64-6.1.1.23I/iosxrv-fullk9-x64-6.1.1.23I.box -v -t -s 6.1.1
[iosxr_store_box.py:141 -                 main()] Input box is: 'machines/iosxrv-fullk9-x64-6.1.1.23I/iosxrv-fullk9-x64-6.1.1.23I.box'
[iosxr_store_box.py:142 -                 main()] Message is:   'No reason for update specified'
[iosxr_store_box.py:143 -                 main()] Sender is:    'rwellum@cisco.com'
[iosxr_store_box.py:144 -                 main()] Receiver is:  'xr64-updates@cisco.com'
[iosxr_store_box.py:145 -                 main()] Release is:   'False'
[iosxr_store_box.py:146 -                 main()] Test Only is: 'True'
[iosxr_store_box.py:147 -                 main()] Sub dir is:   '6.1.1'
[iosxr_store_box.py:50 -        generate_hash()] SHA256: 4bba6e811494beff87d7fcdc164d895463842e2df2df76d0cfa79563077f510a
[iosxr_store_box.py:56 -        generate_hash()] Hash file is iosxrv-fullk9-x64-6.1.1.23I.box.sha256.txt